src/test/java/api/loans/scenarios/ChangeDueDateAPITests.java

```diff
@@ -171,6 +172,22 @@ public void cannotChangeDueDateWhenClaimedReturned() {
       hasUUIDParameter("itemId", item.getId()))));
   }
 
+  @Test
+  public void cannotChangeDueDateWhenItemIsAgedToLost() {
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    val ageToLostResult = ageToLostFixture.createAgedToLostLoan();
+
+    final Response response = changeDueDateFixture.attemptChangeDueDate(
+      new ChangeDueDateRequestBuilder()
+        .forLoan(ageToLostResult.getLoanId())
+        .withDueDate(newDueDate));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("item is Aged to lost"),
+      hasUUIDParameter("itemId", ageToLostResult.getItemId()))));
+  }
```

Comment:
Can these similar API tests be consolidated and replaced with unit tests, like in other recent pull requests?

Reply:
Yes, they are replaced with unit tests now.